tools/ExtensionsMetadataGenerator/src/ExtensionsMetadataGenerator.Console/ExtensionsMetadataGenerator.Console.csproj

```diff
@@ -9,9 +9,11 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects> 
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
+    <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)..\..\..\..\src.ruleset</CodeAnalysisRuleSet>
```

Comment:
StyleCop had a lot of warnings in this solution. I went through and got them all resolved.

Reply:
Just looked. The projects were always referencing StyleCop.Analyzers, so maybe all those warnings were just always there? I suspect the common.props must have turned them into errors, though... but now that I've cleaned them all up, I'll leave them.